dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/UserService.java

```diff
@@ -1,382 +1,390 @@
-package org.hisp.dhis.user;
-
-/*

```

Comment:
See if you can avoid reformatting (?) the code so that we can get a clean diff which is easy to inspect and back-port to previous versions. Applies to a few more classes below.